frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-content-details.tsx

```diff
@@ -39,38 +41,38 @@ const Details: React.FC<DetailsProps> = ({ obj }) => {
           <dl className="co-m-pane__details">
             {size && (
               <>
-                <dt>Size</dt>
+                <dt>{t('volume-snapshot-content~Size')}</dt>
                 <dd>{sizeMetrics}</dd>
               </>
             )}
-            <dt>Volume Snapshot</dt>
+            <dt>{VolumeSnapshotModel.label}</dt>
```

Comment:
It's better to externalize this string. We might still want to translate it even if it's a k8s resource.

Reply:
updated